src/databases.py

```diff
@@ -137,9 +114,7 @@ def get_uris(self) -> typing.Dict[str, str]:
             # https://github.com/canonical/charm-relation-interfaces/blob/main/interfaces/mysql_client/v0/schemas/provider.json
             if not all(data.get(key) for key in ("endpoints", "username", "password")):
                 logger.warning("Incorrect relation data from the data provider: %s", data)
-                raise InvalidDatabaseRelationDataError(
-                    f"Incorrect relation data from the data provider: {data}"
-                )
+                continue
```

Comment:
Why the error is not being raised anymore?
If an application needs database to start and there is no correct relation data, what would happen?

Reply:
In that case, the charm will enter block status with the error message `Webserver configuration check failed, please review your charm configuration or database relation`